apps/AEPSampleAppNewArchEnabled/app/+html.tsx

```diff
@@ -0,0 +1,39 @@
+import { ScrollViewStyleReset } from 'expo-router/html';
+import { type PropsWithChildren } from 'react';
+
+/**
+ * This file is web-only and used to configure the root HTML for every web page during static rendering.
+ * The contents of this function only run in Node.js environments and do not have access to the DOM or browser APIs.
```

Comment:
Can we remove this file since it's "web-only"?

Reply:
removed